maps/map_files/BigRed/BigRed.dmm

```diff
@@ -34454,6 +34514,26 @@
 "rpc" = (
 /turf/open/floor,
 /area/bigredv2/outside/telecomm/security)
+"rpd" = (
+/obj/item/storage/toolbox/mechanical{
+	pixel_x = -5;
+	pixel_y = 16
+	},
+/obj/structure{
```

Comment:
this should be its own defined prop as a child of props/invuln or similar. No more instanced props like this.